modules/tinymce/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Added a new `editor.options` API to replace the old `editor.settings` and `editor.getParam` APIs #TINY-8206
 - The `tabindex` attribute is now copied from the target element to the iframe #TINY-8315
+- Added new `AutocompleterStart/AutocompleterUpdate/AutocompleterEnd` events #TINY-8279
+- Added new `CloseAutocompleter/ReloadAutocompleter` commands #TINY-8279
```

Comment:
Wasn't sure if we removed the `mce` prefix or not read some where in notion that we should drop it I guess that was changed after I read that though. Also I don't want to call these `mceAutocompleterUpdate` and `mceAutocompleterEnd` since those events with similar names can fire independently of these commands and they might fire more events in the future like separating the end reason etc.

Reply:
The `mce` prefix change was pushed out of 6.0 to my knowledge, or at least that's what @metricjs had mentioned to us.